tests/test_base.py

```diff
@@ -152,6 +165,17 @@ def testDotflnRelative(self):
         finally:
             os.chdir(wd)
 
+    def testDotflnTilde(self):
+        """Checks DOT_FLN with no available environment variables"""
+        if 'SPACEPY' in os.environ:
+            del os.environ['SPACEPY']
+        if 'HOME' in os.environ:
+            del os.environ['HOME']
+        if 'HOMEDRIVE' in os.environ:
+            del os.environ['HOMEDRIVE']
```

Comment:
This makes me a little nervous about affecting other tests down the line. Can we save/restore these in a try/finally block?